Makefile.am

```diff
@@ -53,3 +53,37 @@ endif
 	@echo "For more information please see:"
 	@echo "  https://suricata.readthedocs.io/en/latest/rule-management/index.html"
 	@echo ""
+
+#--- clang-formatting ---
+# Check if branch changes formatting is correct
+.PHONY: check-style-branch
```

Comment:
These targets help with github actions. More importantly, they help with reformatting code and whole branches.

The makefile targets, as well as the underlying script that gets called, also isolate the dev from knowing which git-flang-format version to call (on ubuntu).

Reply:
Nope, they do not matter for the actions, as that is all scripted. In fact I no longer use make as it 
hides the scripts exit code" there.

I only added the make targets as an option and alternative to use for people essentially. The idea was that it might be easier for people to remember some of the more complicated calls such as ``make diffstat-style-branch`` than ``./scripts/clang-format.sh check-branch --make --diffstat --show-commits``. Having said that, for the ones that mostly count for devs, there is not much difference directly calling the script.

Some projects like having that option, some not. I am usually in the latter part to be honest, but wanted to give you guys an option. It can easily be removed if not needed.

Reply:
Any quick yay or nay on whether to keep the make targets? I get the feeling from @jasonish they should rather be removed?

Reply:
I removed the make targets for v5. Agree with @jasonish that they are not that useful